src/Auth/BaseAuthenticate.php

```diff
@@ -37,6 +37,7 @@
      * - `userModel` The alias for users table, defaults to Users.
      * - `scope` Additional conditions to use when looking up and authenticating users,
      *    i.e. `['Users.is_active' => 1].`
+     * - `finder` The finder method to use to fetch user record. Defaults to 'all'.
```

Comment:
Yup I had thought that too. Though we should skip triggering errors like we normally do for deprecated features. Will just undocument those options and mention they are deprecated in migration guide.